### PR TITLE
Get-dbadatabase status improvements

### PR DESCRIPTION
**Pull request by Stuart-Moore**
Fixes # 

Changes proposed in this pull request:
 - Status parameter now works with comma combined statuses (express is bad for this ie; "normal, AutoClosed")
 - Allows multiple statuses to be filtered for. Wanted this for the continue restore functionality to get dbs in states that will allow more log restores (recovering, standby)
 - No other changes

How to test this code: 
- [ ] Get-DbaDatabase -SqlInstance localhost\sqlexpress2016 -Status standby,recovery
- [ ] 

Has been tested on minimum requirements:
- [ ]  Powershell 3
- [ ]  Windows 7
- [ ]  SQL Server 2000

Has been tested on maximum requirements:
- [ ]  SQL Server vNext
- [ ]  Windows 10
- [ ]  Azure Database

Tests for tester:
- [x] Working/useful help content, including link to command on dbatools web site
- [x] All examples work as advertised
- [x] Does not contain template content
- [x] Does not contain excessive/unnecessary amounts of comments
- [x] Works remotely
- [x] Works locally
- [x] Works on lower versions or throws error specifying version not supported
- [x] Works with named instances
- [x] Works with clustered instances
- [x] Handles offline/read only databases
- [x] Supports multiple servers (at the command line or piped from Get-SqlRegisteredServerName)
- [x] No un-handled errors which stop the command working with multiple servers

### CODE REVIEW

File: functions/Get-DbaDatabase.ps1 [hunk 1]
```diff
@@ -99,8 +99,7 @@ Returns databases on multiple instances piped into the function
         [parameter(ParameterSetName = "Encrypted")]
         [switch]$Encrypted,
         [parameter(ParameterSetName = "Status")]
-        [ValidateSet('EmergencyMode', 'Normal', 'Offline', 'Recovering', 'Restoring', 'Standby', 'Suspect')]
-        [string]$Status,
```
Comment (potatoqualitee) [inline functions/Get-DbaDatabase.ps1]: Is it possible to keep the validateset? It helps with tabbing. We could still keep the string[]

Comment (Stuart-Moore) [inline functions/Get-DbaDatabase.ps1]: It doesn't like that :( :
image

advice appears to be to user DynamicParameters, which are on the way out?

or enums? Are you happy they'd work far enough back?

Comment (potatoqualitee) [inline functions/Get-DbaDatabase.ps1]: DynamicParams are not on the way out, we're just doing them differently using TEPP, but I believe the answer would be 

```
[ValidateSet('EmergencyMode', 'Normal', 'Offline', 'Recovering', 'Restoring', 'Standby', 'Suspect')]
[string[]]$Status
```

Comment (Stuart-Moore) [inline functions/Get-DbaDatabase.ps1]: Apologies, something strange was going on in my session. That error was what I was getting with ValidateSet

Bounced box and it works now. Will push up

**Comment by potatoqualitee** [comment]
sweet, thank you!

**Pull request merged**